tests/extensions/ccd001-direct-execute.test.ts

```diff
@@ -1,15 +1,14 @@
 import { Account, assertEquals, Clarinet, Chain } from "../../utils/deps.ts";
-import { BASE_DAO, EXTENSIONS, PROPOSALS } from "../../utils/common.ts";
+import { EXTENSIONS, PROPOSALS } from "../../utils/common.ts";
 import { CCD001DirectExecute } from "../../models/extensions/ccd001-direct-execute.model.ts";
 
-const ccd001DirectExecute = new CCD001DirectExecute();
-
```

Comment:
👍

@whoabuddy, with the forthcoming fuzz/prop stuff the following will be also possible:

![image](https://user-images.githubusercontent.com/287532/200809879-2b62c38b-c28c-4ba1-949c-68c3bead5621.png)